src/EditorFeatures/Core.Cocoa/PreviewPane/PreviewPane.cs

```diff
@@ -6,51 +6,147 @@
 
 using System;
 using System.Collections.Generic;
+using System.Text;
 using AppKit;
+using CoreGraphics;
+using Foundation;
 using Microsoft.CodeAnalysis.Editor.Implementation.Preview;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.VisualStudio.Text.Editor;
 
-#pragma warning disable IDE0052 // Remove unread private members
+#pragma warning disable IDE0060 //Remove unused parameter
```

Comment:
Which parameter is this for? Is it an analyzer false positive, or a valid complaint? It looks like everything is used here.

Reply:
In constructor optionPageGuid, logIdVerbatimInTelemetry...

Reply:
Before they were set to field but not anymore... Hence different warning...

Reply:
Oh right, yes of course. Do we need to keep them around, or are you just avoiding breaking VS for Mac?

I'm asking because of https://github.com/dotnet/roslyn/issues/48871 - we'd like to clean all of these up if we can, slowly.

Reply:
Its called from this project so it wouldn't break, its just that goal is to use all this in future, and removing it would need to delete code on PreviewPaneService.cs that we would have to re-add later...